.travis.yml

```diff
@@ -74,7 +74,8 @@ install:
 - source scripts/setup.sh || exit 1
 - source scripts/setup_gae.sh || exit 1
 
-script:
+bundler_args: --retry 1
```

Comment:
the whole script section or just the bundler_args retry?

On Thu, Jul 28, 2016 at 3:19 PM, Barnabas Makonda <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/oppia/oppia/pull/2237#discussion_r72685031>:
>
> > @@ -74,7 +74,8 @@ install:
> >  - source scripts/setup.sh || exit 1
> >  - source scripts/setup_gae.sh || exit 1
> >
> > -script:
> > +bundler_args: --retry 1
>
> sorry one thing can you put this before before_install
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/oppia/oppia/pull/2237/files/c87076de023f037b5e53f7222cd19c00d83d7310#r72685031>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGIsJDLVNA-uch0dcKFbXxln7ES7qoGTks5qaQDEgaJpZM4JHrA1>
> .
>